aws-analytics-pinpoint/src/main/java/com/amplifyframework/analytics/pinpoint/PinpointManager.kt

```diff
@@ -38,35 +38,37 @@ internal class PinpointManager constructor(
 
     val analyticsClient: AnalyticsClient
     val sessionClient: SessionClient
-    private val targetingClient: TargetingClient
-    private val sdkName = "" // TODO: confirm whether to use amplify-android or aws-sdk-android
+    val targetingClient: TargetingClient
+    private val sdkName = "amplify-android" // TODO: confirm whether to use amplify-android or aws-sdk-android
 
     init {
         val pinpointClient = PinpointClient {
             credentialsProvider = this@PinpointManager.credentialsProvider
             region = awsPinpointConfiguration.region
         }
         val pinpointDatabase = PinpointDatabase(context)
-        sessionClient = SessionClient(context, null)
         val sharedPrefs =
             context.getSharedPreferences(awsPinpointConfiguration.appId, Context.MODE_PRIVATE)
+        val sharedPrefsUniqueIdService = SharedPrefsUniqueIdService(sharedPrefs)
         val androidAppDetails = AndroidAppDetails(context, awsPinpointConfiguration.appId)
         val androidDeviceDetails = AndroidDeviceDetails(getCarrier(context))
         targetingClient = TargetingClient(
             pinpointClient,
             PinpointNotificationClient(),
-            SharedPrefsUniqueIdService(sharedPrefs),
+            sharedPrefsUniqueIdService,
             sharedPrefs,
             androidAppDetails,
             androidDeviceDetails,
             context
         )
+        sessionClient = SessionClient(context, null, targetingClient, sharedPrefsUniqueIdService)
```

Comment:
Is the second param in SessionClient used anywhere else in production code? I see its null here.
Would be best to provide default value of null and used named parameters for readability.